hudi-common/src/main/java/org/apache/hudi/common/table/timeline/dto/DataFileDTO.java

```diff
@@ -36,22 +36,22 @@
   @JsonProperty("fileLen")
   private long fileLen;
 
-  public static HoodieDataFile toHoodieDataFile(DataFileDTO dto) {
+  public static HoodieBaseFile toHoodieDataFile(DataFileDTO dto) {
```

Comment:
@vinothchandar Do we want to rename the method as well to 'toHoodieBaseFile' ?

Reply:
+1